### PR TITLE
🏗 Add OWNERS.yaml for visual diffs

### PR DESCRIPTION
**Pull request by danielrozenberg**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@rsimha good point

File: examples/visual-tests/OWNERS.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- danielrozenberg
+- erwinm
+- esth
```
Comment (estherkim) [inline examples/visual-tests/OWNERS.yaml]: my github username is estherkim 8)

Comment (danielrozenberg) [reply examples/visual-tests/OWNERS.yaml]: And Erwin's is @erwinmombay :roll_eyes:

**Pull request merged**